.changeset/eight-dolphins-accept.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-kubernetes': patch
+'@backstage/plugin-kubernetes-backend': patch
+'@backstage/plugin-kubernetes-common': patch
+---
+
+Add new backend endpoints, service/:serviceId endpoint is deprecated, backend now requires discovery api config
```

Comment:
Thinking it's worth splitting this up into separate changesets. Also since I see "now requires" in there I assume that should be a breaking `minor` bump? x)

Reply:
Makes sense will make it a minor bump

Reply:
fixed